security_monitoring/datadog_checks/__init__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2021-present
```

Comment:
Can we safely remove this folder?

Reply:
We can't remove the entire module since it contains some data that's used (e.g., the version or a dummy check), but we could remove the `config_models` submodule. It's gone now.